tensorflow/lite/micro/tools/make/targets/xtensa_makefile.inc

```diff
@@ -34,6 +34,8 @@ endif
 #   TARGET_ARCH=hifi4 --> -DHIFI4
 TARGET_ARCH_DEFINES := -D$(shell echo $(TARGET_ARCH) | tr [a-z] [A-Z])
 
+THIRD_PARTY_C_OPTIMIZATION_LEVEL := -O3 -LNO:simd
```

Comment:
let's keep this change for a separate PR

Reply:
Rolled back in commit https://github.com/tensorflow/tflite-micro/pull/774/commits/90c7ba043d30dc2879e5bb82e8e62c34f0be4418

and created bug: 

http://b/210160638